src/scripts/install-chromedriver.sh

```diff
@@ -157,7 +157,8 @@ else
   echo $MATCHING_CHROMEDRIVER_URL_RESPONSE
   if [[ $MATCHING_CHROMEDRIVER_URL_RESPONSE == 404 ]]; then
     echo "Matching Chrome Driver Version 404'd, falling back to first matching major version."
-    CHROMEDRIVER_VERSION=$( curl https://googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone.json | grep -o "$CHROME_VERSION_MAJOR.*" | grep -o "version.*" | grep -o '\:*'"$CHROME_VERSION_MAJOR"'.*,' | sed 's/".*//')
+    CHROME_VERSION_MAJOR=\"${CHROME_VERSION_MAJOR}\"
+    CHROMEDRIVER_VERSION=$( curl https://googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone.json | grep -o "$CHROME_VERSION_MAJOR.*" | grep -o "version.*" | sed 's/version":"//' | sed 's/".*//')
```

Comment:
I take it this doesn't have access to `jq`?

Reply:
It does have access to jq, but every time I tried using it previously with this json I had issues with it outright not working, so I reverted back to `grep` and `sed` as thats where I am much more experienced

Reply:
I think something like this would work 

```
curl https://googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone.json | jq ".milestones.\"116\".version"
```

Reply:
I just tried that @karlentwistle and it works perfectly thank you, I guess in my hurry I overlooked the double quotes in the json

Reply:
trying it now, with $CHROME_MAJOR_VERSION instead of `116` if that works it'll be the solution we go with.